docs/learn/learn-guides-treasury.md

```diff
@@ -109,6 +125,134 @@ in this document.
+
+The amount should be simply the amount of USDT, where each `1` USDT is `1000000`. Because we are
+asking for 100 USDT, we put `100000000` as the input for the amount:
+
+### Specifying beneficiary
+
+The beneficiary account will be one on AssetHub. For this reason, the XCM junction must be specified
+as follows, with one junction (`X1`) and the beneficiary account(`AccountID32`), which is an
+address:
+
+### Specifying `validFrom` (optional)
+
+The `validFrom` field is optional, and refers to the block height upon which the payout may be
+issued. For more information on this field, refer to the
+[guide below.](#creating-a-staged-proposal-with-validfrom)
+
+### Summary: Final Call
+
+The final call should look like the following, where we:
+
+- Specify our asset as **USDT** on **AssetHub**.
+- Specify the **amount** of **USDT** (100).
+- Specify the beneficiary address.
+- If applicable, use the `validFrom` field to specify a block number upon which the payout becomes
+  valid.
+
+The next steps are to:
+
+- Copy the preimage (and its length)
+- Sign and submit the preimage
+
+Once this is finished, one may submit a proposal
+[as stated above](#submit-a-treasury-track-referendum). Keep in mind one will also need to
+[provide the decision deposit as well!](#place-a-decision-deposit-for-the-treasury-track-referendum)
+
+## Creating a Staged Proposal with `validFrom`
+
+Staged proposals are similar to a tranche or milestone-based funding model; instead of all spends
+being paid simultaneously, each portion is redeemable at a certain block height. This is done by
+specifying each milestone, set at its respective amount and block height at which it becomes
+redeemable.
+
+For example, take the following "staged" proposal which has three milestones, each at 100 DOT, and
+is redeemable at the following block heights. Usually, block heights correspond to a date in the
+future:
+
+- 100 DOT paid out at block height 1000
+- 200 DOT paid out at block height 2000
+- 500 DOT paid out at block height 4000
+
+Governance can also propose to **void** a staged proposal before it has completed all of its
+pay-outs.
+
+:::info Calculating dates from block heights
+
+Although not the most accurate form of measurement, block heights can be used to correspond to
+certain dates in the future.
+
+:::
+
+### Using `batchAll` for multi-spend proposals
+
+Using the utility pallet, one can batch multiple `treasury.spend` calls together. Each of these
+calls can have their own `validFrom` block heights, which allows for scenarios such as the above to
+be possible.
+
+### Using `validFrom` for a milestone-based proposal
+
+Once each spend is defined within the batched call, the validFrom field can be utilized to specify
+the "date", or block height, at which each spend will be executed.
+
+### Manually claiming payouts
+
+:::info Spends can expire!
+
+Keep in mind that once the `validFrom` block height has been reached, you will have to claim the
+spend before it expires.
```

Comment:
Can you add more info about this? Before you said that this doesn't face spending period limitations, but this implies that there are some temporal limitations?

Reply:
30 days according to this writeup https://forum.polkadot.network/t/multi-asset-treasury-and-milestone-based-spends/6780